pkg/kubelet/cm/deviceplugin/manager_test.go

```diff
@@ -261,6 +261,7 @@ func TestCheckpoint(t *testing.T) {
 	resourceName2 := "domain2.com/resource2"
 
 	testManager := &ManagerImpl{
+		socketdir:        "/tmp",
```

Comment:
It's not guranteed that we have a "/tmp" dir.
We uaussly use `ioutil.TempDir`.

It would be better to remove the file explicitly.